guides/20_Builders_Journey.md

```diff
@@ -0,0 +1,476 @@
+| Messaging            | [Blog](https://aws.amazon.com/blogs/business-productivity/build-chat-applications-in-ios-and-android-with-amazon-chime-sdk-messaging/)                                                                                                                                                                            |
+| Android\*            | Meetings                                                                                                                                                                                                                                                                                                          | [iOS and Android Overview](https://aws.amazon.com/blogs/business-productivity/amazon-chime-sdks-ios-android/)[Blog](https://aws.amazon.com/blogs/business-productivity/building-a-meeting-application-on-android-using-the-amazon-chime-sdk/)                                                        |
+| Messaging            | [Blog](https://aws.amazon.com/blogs/business-productivity/build-chat-applications-in-ios-and-android-with-amazon-chime-sdk-messaging/)                                                                                                                                                                            |
+| PSTN / SMA           | Inbound Calling                                                                                                                                                                                                                                                                                                   | [Blog](https://github.com/aws-samples/amazon-chime-sma-update-call)[Demo Code](https://github.com/aws-samples/amazon-chime-sma-update-call)                                                                                                                                                          |
+
+\*Requires access to compatible iOS or Android hardware
+
+## 3.0 Get ready for a Proof of Concept (POC)
+
+According to the Meriam Webster Dictionary a Proof of Concept is "something that demonstrates the feasibility of a concept". An Amazon Chime SDK POC is usually designed to help validate that core goals can be met using the Amazon Chime SDK. It is not generally a full-blown application and is not usually deployed to the masses but is tested and verified by a small subset of folks, often just within IT.
+
+###
+
+### 3.1 What is a POC Readiness Review?
+
+It is important that you what your goals are. You document what they are trying to accomplish in the POC in the POC Readiness Review. In this review, you and AWS review the timelines, success criteria, and expectations.
+
+### 3.2 POC Readiness Review
```

Comment:
Should we also tell builders how they can create this ticket if needed or what would be the process for engaging with the SAs?